package.json

```diff
@@ -123,11 +124,12 @@
     "precise-commits": "^1.0.2",
     "prettier": "1.18.2",
     "react-test-renderer": "^16.8.6",
+    "regenerator-runtime": "^0.13.3",
     "require-context.macro": "^1.0.4",
     "shelljs": "^0.8.3",
     "storybook-readme": "^5.0.5",
     "typedoc": "^0.16.9",
-    "typescript": "^3.5.2",
+    "typescript": "3.7.5",
```

Comment:
Typescript 3.8 introduces a breaking change to our type-check. https://github.com/microsoft/TypeScript/issues/36917